osx_run.sh

```diff
@@ -1,53 +1,7 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -euf -o pipefail
 set -a
 
-export NODE_ENV=development
-export DEBUG=True
-export PORT=8079
-export COVERAGE_DIR=htmlcov
-export DATABASE_URL=postgres://postgres:postgres@localhost:5432/postgres
-export MICROMASTERS_USE_WEBPACK_DEV_SERVER=True
-export MICROMASTERS_SECURE_SSL_REDIRECT=False
-export MICROMASTERS_DB_DISABLE_SSL=True
+. ./_osx_setup.sh
```

Comment:
does this mean that `osx_run.sh` will do `npm install`, `pip install`, etc every time? it seems like that would make the command slow unnecessarily.

am I missing something, or is there a reason why `osx_run.sh` couldn't just export the necessary environment variables, source `.env`, and then run the command?